src/services/deleteNoteService.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const deleteNoteService = (id, token) => {
```

Comment:
I guess if we can create one file "noteService.js" and add all the api calls (like addNote, updateNote etc )related to note functionality that will just help to keep related things at one place. That's my personal preference. 
In e-commerce I followed that, for ex: had service files cartService (consisting of addToCart api, removeFromCart api etc). Just a suggestion.

Reply:
Thank you for this suggestion! 
I have added all the services into separate folders. I shall also add them in a single file before creating the next feature.